crawler/rabbit/async_consumer.py

```diff
@@ -58,7 +85,9 @@ def connect(self):
             credentials=credentials,
         )
         if self._server_details.uses_ssl:
-            connection_params.ssl_options = SSLOptions(create_default_context())
+            cafile = os.getenv("REQUESTS_CA_BUNDLE")
+            ssl_context = ssl.create_default_context(cafile=cafile)
```

Comment:
This is the part that gives it the CA certificate file.  If `getenv` above returns `None` because the environment variable is not set, then the SSL context will be a standard one like in the removed line above.